docs/content/Installation/install-with-ansible/prerequisites.md

```diff
@@ -212,9 +212,12 @@ sets of variables cannot be used at the same time.
 | `pgo_operator_namespace`          |             | **Required** | Set to configure the namespace where Operator will be deployed.                                                                                                                  |
 | `pgo_tls_ca_store`                |             |          | Set to add additional Certificate Authorities for Operator to trust (PEM-encoded file).                                                                                      |
 | `pgo_tls_no_verify`               | false       |          | Set to configure Operator to verify TLS certificates.                                                                                                                            |
-| `pgo_client_container_install` | false | | Installs the pgo-client deployment along with ansible isnstall |
+| `pgo_client_container_install` | false | | Installs the pgo-client deployment along with ansible install |
 | `pgo_apiserver_url` | `https://postgres-operator` | | Sets the `pgo_apiserver_url` in the pgo-client deployment |
 | `pgo_client_cert_secret` | `pgo.tls` | | Sets the secret that the pgo-client will use when connecting to the operator. Secret should hold the TLS certs |
+| `pod_anti_affinity`               | preferred   |           | Sets the default pod anti-affinity for the deployed PostgreSQL clusters, which is applied to the PostgreSQL instances, the pgBackRest repository, and any pgBouncer instances |
+| `pod_anti_affinity_pgbackrest`    |             |           | If set, overrides the value of `pod_anti_affinity` for just the pgBackRest repository and sets the anti-affinity for the pgBackRest repository compared to its pod anti-affinity group |
```

Comment:
Wording here is a bit confusing, specifically where is says:
`...sets the anti-affinity for the pgBackRest repository compared to its pod anti-affinity group`.

Maybe just leaving the first part is sufficient, i.e. just including the following without any implementation details:
`If set, overrides the value of `pod_anti_affinity` for just the pgBackRest repository.`

(we could do the same for `pod_anti_affinity_pgbouncer` below as well)